server/channels/db/migrations/migrations.list

```diff
@@ -210,10 +210,16 @@ channels/db/migrations/mysql/000104_upgrade_notifyadmin.down.sql
 channels/db/migrations/mysql/000104_upgrade_notifyadmin.up.sql
 channels/db/migrations/mysql/000105_remove_tokens.down.sql
 channels/db/migrations/mysql/000105_remove_tokens.up.sql
+<<<<<<< HEAD
 channels/db/migrations/mysql/000106_fileinfo_channelid.down.sql
 channels/db/migrations/mysql/000106_fileinfo_channelid.up.sql
 channels/db/migrations/mysql/000107_threadmemberships_cleanup.down.sql
 channels/db/migrations/mysql/000107_threadmemberships_cleanup.up.sql
+||||||| 56b18ca7bf (MM-45956: Optimize FileInfo stats query (#22603))
+channels/db/migrations/mysql/000106_fileinfo_channelid.down.sql
+channels/db/migrations/mysql/000106_fileinfo_channelid.up.sql
+=======
+>>>>>>> parent of 56b18ca7bf (MM-45956: Optimize FileInfo stats query (#22603))
```

Comment:
Leftovers from some merge conflicts.

Reply:
Ooh, good catch! I'm actually curious: what do we use this file for?

Reply:
That's a good question.